tests/test_merge.py

```diff
@@ -26,3 +27,54 @@
 )
 def test_get_average_frame_count(samples: str, count: float) -> None:
     assert get_average_frame_count(samples) == count
+
+
+@pytest.mark.parametrize("insert_dso_name", [False, True])
+@pytest.mark.parametrize(
+    "stack, outcome_dict",
```

Comment:
Let's add at least one test which checks for multiple frames in a stack? (i.e multiline input)

Reply:
👍